content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -88,11 +137,30 @@ npm install @opentelemetry/auto-instrumentations-node
 
 The tracing setup and configuration should be run before your application code. One tool commonly used for this task is the [`-r, --require module`](https://nodejs.org/api/cli.html#cli_r_require_module) flag.
 
-Create a file with a name like `tracing.js` which will contain your tracing setup code.
+Create a file named `tracing.ts|js`, which will contain your tracing setup code.
 
-```javascript
-/* tracing.js */
+{{< ot-tabs TypeScript JavaScript >}}
 
+{{< ot-tab >}}
+/*tracing.ts*/
+// Require dependencies
+import * as opentelemetry from "@opentelemetry/sdk-node";
+import { getNodeAutoInstrumentations } from "@opentelemetry/auto-instrumentations-node";
+import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api';
+
+// For troubleshooting, set the log level to DiagLogLevel.DEBUG
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO);
+
+const sdk: any = new opentelemetry.NodeSDK({
```

Comment:
I don't think we need to introduce a TypeScript  dependency int his Getting Start. As far as I can see the `: any` is the only TypeScript typing in this code snippet. I think using vanilla Node/JavaScript makes it easier to get things going

Reply:
Good point, and I would tend to agree.
Comments @open-telemetry/javascript-approvers @svrnm @cartermp?

Reply:
I think it's fine to just rely on type inference here for those code sample, but I disagree with not having TypeScript as a way to get started. There's a ton of TS developers who don't use vanilla JS anymore.

Reply:
So, what's the final call here? @svrnm - any thoughts/preferences?

Reply:
I vote to keep for now - we can adjust this after the fact if we need to.

(also this is for the typescript tab, not the JS tab, so there's no real harm here)

Reply:
It was the whole purpose of this PR to have TS & JS side-by-side, I agree that the added value is not big since most code looks almost identically, but from an end user perspective it's sometimes just about "let me copy & paste that so I can get started quickly" and that's given that way.

Reply:
@svrnm so nothing to update right?